zcash_client_sqlite/src/wallet/init/migrations/transparent_gap_limit_handling.rs

```diff
@@ -480,7 +480,7 @@ impl<P: consensus::Parameters, C: Clock, R: RngCore> RusqliteMigration for Migra
                                 .expect("previously generated an address"),
                         )
                         .expect("roundtrip");
-                    let ua = uivk.address(di, UnifiedAddressRequest::AllAvailableKeys)?;
+                    let ua = uivk.address(di, UnifiedAddressRequest::ALLOW_ALL)?;
```

Comment:
This change is safe, because the migration will either have succeeded with `AllAvailableKeys`, or it will have failed in which case the migration will not have been applied and will re-run with the correction the next time the wallet starts up.

Reply:
This change is also safe because it is only used in the context of diversifiers derived from either Sapling or Orchard receivers, ensuring that the resulting UA will contain a shielded receiver.